sql/world_updates/2017-01-29_01_scripttext.sql

```diff
@@ -0,0 +1,31 @@
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8773', 'What are you attacking him for? The dangerous one is over there,fool!', '36477', '14', '16933');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8774', 'Enough moving around! Hold still while I blow them all up!', '36477', '14', '16929');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8775', 'Wait! Stop! Don't kill me, please! I'll tell you everything!', '36477', '14', '16934');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8776', 'I'm not so naive as to believe your appeal for clemency, but I will listen.', '36993', '14', '16611');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8777', 'Why should the Banshee Queen spare your miserable life?', '36990', '14', '17033');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8778', 'What you seek is in the master's lair, but you must destroy Tyrannus to gain entry. Within the Halls of Reflection you will find Frostmourne. It... it holds the truth.', '36477', '14', '16935');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8779', 'Frostmourne lies unguarded? Impossible!', '36993', '14', '16612');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8780', 'Frostmourne? The Lich King is never without his blade! If you are lying to me...', '36990', '14', '17034');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8781', 'I swear it is true! Please, don't kill me!!', '36477', '14', '16936');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8782', 'Worthless gnat! Death is all that awaits you!', '36477', '14', '16753');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8783', 'Urg... no!!', '36477', '14', '16937');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8784', 'Do not think that I shall permit you entry into my master's sanctum so easily. Pursue me if you dare.', '36477', '14', '16754');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8785', 'What a cruel end. Come, heroes. We must see if the gnome's story is true. If we can separate Arthas from Frostmourne, we might have a chance at stopping him.', '36993', '14', '16613');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8786', 'A fitting end for a traitor. Come, we must free the slaves and see what is within the Lich King's chamber for ourselves.', '36990', '14', '17035');
+
```

Comment:
Do not use single inserts, use:
`INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES 
   (values 1),
   (values 2),
   (values 3);`

Check out 216-08-12_01_pet_level_abilities.sql for an detailed example.